### PR TITLE
Set neutron MTU according to actual value

### PR DESCRIPTION
**Pull request by rsafrono**
When deployed on OCP with default MTU settings actual MTU is limited to 1400 while neutron default is 1500. Wrong MTU setting causes failures during some tests execution.

### CODE REVIEW

File: examples/dt/uni06zeta/control-plane/service-values.yaml [hunk 1]
```diff
@@ -116,6 +116,7 @@ data:
       type_drivers = geneve,vlan
       tenant_network_types = geneve,vlan
       extension_drivers = qos,port_security,dns_domain_ports
+      path_mtu = 1400
```
Comment (karelyatin) [inline examples/dt/uni06zeta/control-plane/service-values.yaml:119]: let's hold, i think we would need some routes in network attachments. Temporary can get this fix too if route fixes are not yet landed in ci-framework

**Comment by karelyatin** [comment]
<< When deployed on OCP with default MTU settings actual MTU is limited to 1400 while neutron default is 1500. Wrong MTU setting causes failures during some tests execution.

@rsafrono just to have more context is this needed only for test_mtu_sized_frame test or also for any other test?

**Comment by rsafrono** [comment]
@karelyatin Recently I've hit only issue with test_mtu_sized_frames. But IIRC we've seen in the past also issues with QoS bw_limit tests, where iperf was reporting zero traffic because of the MTU mismatch. Setting path MTU to 1400 allowed iperf to send traffic and made the QoS bw_limit tests passing in that case.

File: examples/dt/uni06zeta/control-plane/service-values.yaml (hunk 1 above)
Comment (abays) [reply examples/dt/uni06zeta/control-plane/service-values.yaml:119]: @karelyatin Do we still need to hold on merging this?

**Comment by karelyatin** [comment]
> @karelyatin Recently I've hit only issue with test_mtu_sized_frames. But IIRC we've seen in the past also issues with QoS bw_limit tests, where iperf was reporting zero traffic because of the MTU mismatch. Setting path MTU to 1400 allowed iperf to send traffic and made the QoS bw_limit tests passing in that case.

@rsafrono ack to go this route for now

File: examples/dt/uni06zeta/control-plane/service-values.yaml (hunk 1 above)
Comment (karelyatin) [reply examples/dt/uni06zeta/control-plane/service-values.yaml:119]: @abays can we get this in

Comment (abays) [reply examples/dt/uni06zeta/control-plane/service-values.yaml:119]: Can we get an ack on whether this has been tested?

Comment (karelyatin) [reply examples/dt/uni06zeta/control-plane/service-values.yaml:119]: @abays full testing on zeta job not done, just the affected tests were tested in zeta environment with this config and worked fine, also we have component jobs running with this config already so should be fine.

**Review by abays — APPROVED** [review]
/lgtm
/approve
